cmd/eskip/args.go

```diff
@@ -38,6 +39,7 @@ const (
 	appendFiltersFlag  = "append"
 	appendFileFlag     = "append-file"
 	prettyFlag         = "pretty"
+	intentStrFlag      = "intent"
```

Comment:
"intent" should be "indent" or "indentation"

Reply:
Thanks. I fixed typo.